server/controllers/reviews.js

```diff
@@ -16,10 +15,22 @@ exports.getAllReviews = asyncHandler(async (req, res) => {
 exports.createReview = asyncHandler(async (req, res) => {
   try {
     const review = await Reviews.create({
+      userId: req.user.id,
```

Comment:
When we are creating a review, won't we need to update `reviewerId` here? As the logged in user is the one creating the review.